setup.py

```diff
@@ -8,10 +8,10 @@ def main():
         long_description=open('README.rst').read(),
         use_scm_version={"write_to": "py/_version.py"},
         setup_requires=["setuptools-scm"],
-        url='http://py.readthedocs.io/',
+        url='https://py.readthedocs.io/',
         license='MIT license',
         platforms=['unix', 'linux', 'osx', 'cygwin', 'win32'],
-        python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
+        python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
```

Comment:
given it's maintenance-only I don't think we should change this metadata (even though we're not testing it)

Reply:
Reverted in 8c867c15!